x-pack/filebeat/module/panw/panos/test/pan_inc_traffic_ietf.log

```diff
@@ -0,0 +1,100 @@
+535 <14>1 2020-03-27T09:57:41+01:00 PA-VM - - - - 1,2020/03/27 09:57:41,015351000043722,TRAFFIC,end,2305,2020/03/27 09:57:41,192.168.2.4,0.0.0.0,0.0.0.0,0.0.0.0,Internet Access,,,ssl,vsys1,Clients,External,ethernet1/3,ethernet1/1,Default,2020/03/27 09:57:41,7603,1,32801,514,57533,514,0x500070,tcp,allow,905,633,272,9,2020/03/27 09:57:27,0,any,0,2825918,0x0,192.168.0.0-192.168.255.255,Germany,0,5,4,tcp-fin,0,0,0,0,,PA-VM,from-policy,,,0,,0,,N/A,0,0,0,0,d0422b3b-8522-4916-8b97-524c32780f90,0,0,,,,,,,
```

Comment:
This looks like RFC6587 framing since it has that leading number which I assume is the length. 

I'm curious how you go about receiving these? Filebeat could receive these directly with the `tcp` input and the [`framing: rfc6587`](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-tcp.html#filebeat-input-tcp-tcp-framing) option. But the module itself isn't setup to allow this.